test/test_LUDOX_protocol.py

```diff
@@ -79,7 +79,8 @@ def test_create_protocol(self):
         measure = protocol.primitive_step('MeasureAbsorbance', samples=c_measure.output_pin('samples'))
 
         protocol.use_value(wavelength_param, measure.input_pin('wavelength'))
-        protocol.add_output('absorbance', measure.output_pin('measurements'))
+        absorbance_param = protocol.add_output('absorbance', sbol3.OM_MEASURE)
+        protocol.use_value(measure.output_pin('measurements'), absorbance_param)
```

Comment:
It might be nice to add a helper function that combines the functionality of both add_output and use_value